librz/bin/format/elf/elf_imports.c

```diff
@@ -57,6 +57,37 @@ static ut64 get_import_addr_mips(ELFOBJ *bin, RzBinElfReloc *rel) {
 	return plt_addr;
 }
 
+/**
+ * \brief Determines and returns the import address for the given relocation
+ * for the Hexagon architecture.
+ *
+ * \param eo The ElfObject.
+ * \param rel The Elf relocation to get the address for.
+ *
+ * \return The import address or UT64_MAX in case of failure.
+ */
+static ut64 get_import_addr_hexagon(ELFOBJ *eo, RzBinElfReloc *rel) {
+	ut64 got_addr = 0;
+
+	if (!Elf_(rz_bin_elf_get_dt_info)(eo, DT_PLTGOT, &got_addr) || got_addr == UT64_MAX) {
+		return UT64_MAX;
+	}
+
+	ut64 plt_addr = get_got_entry(eo, rel);
+	if (plt_addr == UT64_MAX) {
+		return UT64_MAX;
+	}
+
+	const ut64 pos = COMPUTE_PLTGOT_POSITION(rel, got_addr, 0x3);
+
+	switch (rel->type) {
+	default:
+		return UT64_MAX;
```

Comment:
Please add also a debug message here about unsupported relocation type so that users can report it if they meet binary with those unimplemented relocs.